src/client/components/pages/entities/edition-table.js

```diff
@@ -31,16 +31,27 @@ const {
 } = entityHelper;
 const {Button, Table} = bootstrap;
 
-function EditionTableRow({edition}) {
+function EditionTableRow({edition, showCheckboxes, selectedEntities, toggleRow}) {
 	const name = getEntityLabel(edition);
 	const disambiguation = getEntityDisambiguation(edition);
 	const releaseDate = getEditionReleaseDate(edition);
 	const isbn = getISBNOfEdition(edition);
 	const editionFormat = getEditionFormat(edition);
 
+	/* eslint-disable react/jsx-no-bind */
```

Comment:
I wonder if you could do without this.
Would changing `onClick={() => toggleRow(edition.bbid)}` 
to: `onClick={toggleRow.bind(this, edition.bbid)}` do the trick?

Reply:
that fixes that error but shows a different eslint error. :(